src/main/java/smartthings/ratpack/sqs/DefaultSqsService.java

```diff
@@ -35,7 +37,8 @@ public DefaultSqsService(
     @Override
     public void onStart(StartEvent event) throws Exception {
         if (config.isEnabled()) {
-            this.sqs = sqsConfigService.getAmazonSQSAsync(config.getRegion());
+            sqs = sqsConfigService.getAmazonSQSAsync(config.getRegion());
+            Optional.ofNullable(config.getEndpoint()).ifPresent(sqs::setEndpoint);
```

Comment:
@llinder 
Endpoint is optional. In case it isn't provided client will  automatically select amazon server relative to configured aws credentials and region.